packages/office-ui-fabric-react/src/components/List/examples/List.Grid.Example.tsx

```diff
@@ -4,24 +4,16 @@ import { List } from 'office-ui-fabric-react/lib/List';
 import { IRectangle } from 'office-ui-fabric-react/lib/Utilities';
 import { ITheme, getTheme, mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
 import { createListItems, IExampleItem } from '@uifabric/example-data';
+import { useConst } from '@uifabric/react-hooks';
 
-export interface IListGridExampleProps {
-  items?: IExampleItem[];
-}
-
-interface IListGridExampleClassObject {
-  listGridExample: string;
-  listGridExampleTile: string;
-  listGridExampleSizer: string;
-  listGridExamplePadder: string;
-  listGridExampleLabel: string;
-  listGridExampleImage: string;
-}
-
+let columnCount: number;
```

Comment:
These need to go inside the example, and you'll have to find some way (either useCallback/useMemo or refs) to preserve the updates across renders.